scripts/network/workloads/network-client.toml

```diff
@@ -0,0 +1,12 @@
+restart_interval = 10
+workers = 1
+per_core = false
+
+[workload]
+type = "network"
+server = false
+address = [223, 42, 0, 1]
+target_port = 1337
+arrival_rate = 0.1
+departure_rate = 0.1
```

Comment:
Do you need `arrival_rate` and `departure_rate` for both the server and client. The existing workload example has both for the server, but I wonder if that is correct.